backend/src/main/java/ca/bc/gov/backendstartapi/entity/seedlot/SeedlotCollectionMethod.java

```diff
@@ -0,0 +1,63 @@
+package ca.bc.gov.backendstartapi.entity.seedlot;
+
+import ca.bc.gov.backendstartapi.entity.embeddable.AuditInformation;
+import ca.bc.gov.backendstartapi.entity.seedlot.idclass.SeedlotCollectionMethodId;
+import ca.bc.gov.backendstartapi.enums.ConeCollectionMethodEnum;
+import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.IdClass;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.io.Serial;
+import java.io.Serializable;
+import lombok.AccessLevel;
+import lombok.Generated;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+/** The method used for the collection of a lot of seeds. */
+@Entity
+@Table(name = "seedlot_collection_method")
+@IdClass(SeedlotCollectionMethodId.class)
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+public class SeedlotCollectionMethod implements Serializable {
+
+  @Generated @Serial private static final long serialVersionUID = 4440025991282910997L;
+
+  // region Identifier
+  @Id
+  @ManyToOne(optional = false)
```

Comment:
Sorry, I don't understand the comment. Can you elaborate on the concern?

Reply:
Yep. I'm trying to raise the question: when querying a `SeedlotCollectionMethod` from the database, what are the chances that we may already have the correspondent Seedlot record in hands? 

Also about performance, since `Seedlot` is a huge table, I'm not sure we need to fetch it as soon as a `SeedlotCollectionMethod` is fetched.

Reply:
We can't know for sure yet, but I believe it's very likely we'll have already queried the associated seedlot before this entity; in this case, we'll already have it in the first- or second-level cache (when we configure it), so there'll be no call to the database.